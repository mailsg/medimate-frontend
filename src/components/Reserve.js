import React from 'react';
import { Link } from 'react-router-dom';
import styles from '../css/reserve.module.css';

export const Reserve = () => (
  <section className={styles.container}>
    <article>
      <h1>BOOK AN APPOINTMENT WITH OUR DOCTORS</h1>
      <div className={styles.thematic} />
      <p>
        Tell us about a time when you used the working agreements (or the roles)
        that you set up at the beginning of the project,
        and how they helped you work better as a team.
      </p>
      <div className={styles['booking-btns']}>
        <select>
          <option>London</option>
          <option>Mumbai</option>
        </select>

        <Link to="/Reservation">
          <button type="button">Book now</button>
        </Link>

      </div>
    </article>
  </section>
);

export default Reserve;
