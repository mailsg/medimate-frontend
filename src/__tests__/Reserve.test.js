import React from 'react';
import { render } from '@testing-library/react';
import { BrowserRouter } from 'react-router-dom';
import Reserve from '../components/Reserve';

describe('Booking Component', () => {
  test('renders Booking component', () => {
    const { container } = render(
      <BrowserRouter>
        <Reserve />
      </BrowserRouter>,
    );
    expect(container).toMatchSnapshot();
  });
});
