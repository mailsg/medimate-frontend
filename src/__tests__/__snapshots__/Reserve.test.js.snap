// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Booking Component renders Booking component 1`] = `
<div>
  <section
    class="container"
  >
    <article>
      <h1
        class="reserve-header"
      >
        BOOK AN APPOINTMENT WITH OUR DOCTORS
      </h1>
      <div
        class="thematic"
      />
      <p
        class="reserve-body"
      >
        Tell us about a time when you used the working agreements (or the roles) that you set up at the beginning of the project, and how they helped you work better as a team.
      </p>
      <div
        class="booking-btns"
      >
        <a
          href="/reservation"
        >
          <button
            type="button"
          >
            Book now
          </button>
        </a>
      </div>
    </article>
  </section>
</div>
`;
